### ISSUE TITLE
MenuDropdown: Buttons must have discernible text (button-name)

### ISSUE THREAD

**Issue by jcjuett** [salesforce/design-system-react#2016]
https://design-system-react-co-pr-2004.herokuapp.com/?selectedKind=SLDSMenuDropdown&selectedStory=Custom%20Trigger&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Issue: Custom trigger state fails automated aXe test for discernible button name.

HTML: `<button class="slds-button slds-button_neutral ignore-click-Sk5s0KB0E" tabindex="-1" type="button" aria-expanded="false" aria-haspopup="true"><svg aria-hidden="true" class="slds-button__icon"><use xlink:href="/assets/icons/utility-sprite/svg/symbols.svg#settings"></use></svg></button>`

Solution: add `<span class="slds-assistive-text">Custom Trigger</span>` between `</svg>` and `</button>`

**Comment by interactivellama** [comment]
@AnutaK Try this issue out. It's an accessibility bug. This icon in the button is missing text.

https://github.com/salesforce/design-system-react/blob/master/components/menu-dropdown/__examples__/custom-trigger.jsx#L1

**Comment by interactivellama** [comment]
@AnutaK In order to complete V2MOM measures for this year for accessibility passing on all examples, I'm going to steal this item back. Please review the open issues if you would like to pick up another one.
